docker-compose.yml

```diff
@@ -14,5 +14,6 @@ services:
       - "/srv/daemon/config/:/srv/daemon/config/"
       - "/srv/daemon-data/:/srv/daemon-data/"
       - "/tmp/pterodactyl/:/tmp/pterodactyl"
+      - "/etc/timezone:/etc/timezone:ro"
```

Comment:
Why not just set the timezone env (`TZ=`) based on the system timezone? We've perpetually had issues with this bind mounting, its one the reasons I completely removed it from wings.

Reply:
Then it just sounds like a variable people can set themselves. We can just close this